Mathlib/Analysis/SpecialFunctions/OrdinaryHypergeometric.lean

```diff
@@ -0,0 +1,282 @@
+  apply FormalMultilinearSeries.radius_eq_top_of_forall_image_add_eq_zero _ (1 + ck)
+  refine fun _ ↦ ordinaryHypergeometricSeries_eq_zero_of_neg_nat a b c (Or.inr <| Or.inr hc)
+    (by linarith)
+
+/-- An iff variation on `ordinaryHypergeometricSeries_eq_zero_of_nonpos_int` for `[RCLike 𝕂]`. -/
+lemma ordinaryHypergeometricSeries_eq_zero_iff (n : ℕ) :
+    ordinaryHypergeometricSeries 𝔸 a b c n = 0 ↔
+    ∃ k : ℕ, (k = -a ∨ k = -b ∨ k = -c) ∧ k < n := by
+  refine ⟨fun h ↦ ?_, fun zero ↦ ?_⟩
+  · rw [ordinaryHypergeometricSeries,
+      smul_eq_zero (c:=(_ * (Polynomial.eval c (ascPochhammer 𝕂 n))⁻¹))
+      (x:=ContinuousMultilinearMap.mkPiAlgebraFin 𝕂 n 𝔸)] at h
+    cases' h with h hm
+    · simp only [_root_.mul_eq_zero, inv_eq_zero] at h
+      rcases h with ((hn | h) | h) | h
+      · exact False.elim <| Nat.cast_ne_zero.2 (Nat.factorial_ne_zero n) hn
+      all_goals
+        let ⟨kn, hkn, hn⟩ := (ascPochhammer_eval_eq_zero_iff _ _).1 h
+        exact ⟨kn, by tauto, hn⟩
+    · rw [ContinuousMultilinearMap.ext_iff] at hm
+      absurd hm
+      push_neg
+      exact ⟨fun _ ↦ 1, by simp⟩
+  · have ⟨_, h, hn⟩ := zero
+    exact ordinaryHypergeometricSeries_eq_zero_of_neg_nat a b c h hn
+
+theorem ordinaryHypergeometricSeries_succ_norm_div_norm (n : ℕ)
+    (habc : ∀ kn : ℕ, (kn = -a ∨ kn = -b ∨ kn = -c) → n ≤ kn) :
+    ‖ordinaryHypergeometricSeries 𝔸 a b c (n+1)‖ / ‖ordinaryHypergeometricSeries 𝔸 a b c n‖ =
+    ‖a + n‖ * ‖b + n‖ * ‖c + n‖⁻¹ * ‖1 + (n : 𝕂)‖⁻¹ := by
+  simp [ordinaryHypergeometricSeries, factorial_succ, ascPochhammer_succ_eval]
+  rw [norm_smul (x:=ContinuousMultilinearMap.mkPiAlgebraFin 𝕂 (n + 1) 𝔸),
+    norm_smul (x:=ContinuousMultilinearMap.mkPiAlgebraFin 𝕂 n 𝔸)]
+  simp only [norm_mul, norm_inv, ContinuousMultilinearMap.norm_mkPiAlgebraFin, mul_one]
+  have : ‖(n ! : 𝕂)‖⁻¹ * ‖(n : 𝕂) + 1‖⁻¹ * (‖Polynomial.eval a (ascPochhammer 𝕂 n)‖ * ‖a + n‖) *
+    (‖Polynomial.eval b (ascPochhammer 𝕂 n)‖ * ‖b + n‖) *
+    (‖c + n‖⁻¹ * ‖Polynomial.eval c (ascPochhammer 𝕂 n)‖⁻¹) /
+    (‖(n ! : 𝕂)‖⁻¹ * ‖Polynomial.eval a (ascPochhammer 𝕂 n)‖ *
+    ‖Polynomial.eval b (ascPochhammer 𝕂 n)‖ * ‖Polynomial.eval c (ascPochhammer 𝕂 n)‖⁻¹) =
+    ‖(n ! : 𝕂)‖⁻¹⁻¹ * ‖(n ! : 𝕂)‖⁻¹ * ‖Polynomial.eval a (ascPochhammer 𝕂 n)‖ *
+    ‖Polynomial.eval a (ascPochhammer 𝕂 n)‖⁻¹ * ‖Polynomial.eval b (ascPochhammer 𝕂 n)‖ *
+    ‖Polynomial.eval b (ascPochhammer 𝕂 n)‖⁻¹ * ‖Polynomial.eval c (ascPochhammer 𝕂 n)‖⁻¹⁻¹ *
+    ‖Polynomial.eval c (ascPochhammer 𝕂 n)‖⁻¹ * ‖a + n‖ * ‖b + n‖ * ‖c + n‖⁻¹ * ‖1 + (n : 𝕂)‖⁻¹ :=
+    by ring_nf
+  rw [this, inv_inv, inv_inv]
+  repeat rw [DivisionRing.mul_inv_cancel, one_mul]
+  all_goals
+    rw [norm_ne_zero_iff]
+  any_goals
+    apply (ascPochhammer_eval_eq_zero_iff n _).not.2
+    push_neg
+    exact fun kn hkn ↦ habc kn (by tauto)
+  exact cast_ne_zero.2 (factorial_ne_zero n)
+
+private theorem linear_div_tendsto_one_atTop :
+    Tendsto (fun (k:ℕ) ↦ (a + k) / (b + k)) atTop (nhds 1) := by
```

Comment:
If you add `open scoped Topology` higher up then you can use notation for `nhds`:
```suggestion
    Tendsto (fun (k:ℕ) ↦ (a + k) / (b + k)) atTop (𝓝 1) := by
```